tests/semantic/ignore_files.json

```diff
@@ -0,0 +1,542 @@
+    "solidity/test/libsolidity/semanticTests/externalSource/source.sol",
+    "solidity/test/libsolidity/semanticTests/externalSource/source_import.sol",
+    "solidity/test/libsolidity/semanticTests/externalSource/source_import_subdir.sol",
+    "solidity/test/libsolidity/semanticTests/externalSource/source_name_starting_with_dots.sol",
+    "solidity/test/libsolidity/semanticTests/externalSource/source_remapping.sol",
+    "solidity/test/libsolidity/semanticTests/freeFunctions/import.sol",
+    "solidity/test/libsolidity/semanticTests/functionCall/file_level_call_via_module.sol",
+    "solidity/test/libsolidity/semanticTests/inheritance/member_notation_ctor.sol",
+    "solidity/test/libsolidity/semanticTests/libraries/library_address_via_module.sol",
+    "solidity/test/libsolidity/semanticTests/modifiers/access_through_module_name.sol",
+    "solidity/test/libsolidity/semanticTests/multiSource/circular_import.sol",
+    "solidity/test/libsolidity/semanticTests/multiSource/circular_import_2.sol",
+    "solidity/test/libsolidity/semanticTests/multiSource/circular_reimport.sol",
+    "solidity/test/libsolidity/semanticTests/multiSource/circular_reimport_2.sol",
+    "solidity/test/libsolidity/semanticTests/multiSource/free_different_interger_types.sol",
+    "solidity/test/libsolidity/semanticTests/multiSource/free_function_resolution_base_contract.sol",
+    "solidity/test/libsolidity/semanticTests/multiSource/free_function_resolution_override_virtual.sol",
+    "solidity/test/libsolidity/semanticTests/multiSource/free_function_resolution_override_virtual_super.sol",
+    "solidity/test/libsolidity/semanticTests/multiSource/free_function_resolution_override_virtual_transitive.sol",
+    "solidity/test/libsolidity/semanticTests/multiSource/free_function_transitive_import.sol",
+    "solidity/test/libsolidity/semanticTests/multiSource/import.sol",
+    "solidity/test/libsolidity/semanticTests/multiSource/imported_free_function_via_alias.sol",
+    "solidity/test/libsolidity/semanticTests/multiSource/imported_free_function_via_alias_direct_call.sol",
+    "solidity/test/libsolidity/semanticTests/multiSource/reimport_imported_function.sol",
+    "solidity/test/libsolidity/semanticTests/userDefinedValueType/multisource.sol",
+    "solidity/test/libsolidity/semanticTests/userDefinedValueType/multisource_module.sol",
+
+
+    "// uses msg.sender (always 0 on starknet if tx not from another contract)",
+    "solidity/test/libsolidity/semanticTests/functionTypes/mapping_of_functions.sol",
+    "solidity/test/libsolidity/semanticTests/isoltestTesting/account.sol",
+    "solidity/test/libsolidity/semanticTests/state/msg_sender.sol",
+    "solidity/test/libsolidity/semanticTests/various/multi_modifiers.sol",
+    "solidity/test/libsolidity/semanticTests/various/senders_balance.sol",
+    "solidity/test/libsolidity/semanticTests/viaYul/msg_sender.sol",
+
+    "// fallback function used",
```

Comment:
Why is that a problem?.. Do they really not pass?

Reply:
Not sure, the ignore file has been touched by everyone at this point @BitBaseBit, @JorikSchellekens do you why these fail?

Reply:
We can add those back